pkg/testdata/testdata.go

```diff
@@ -0,0 +1,148 @@
+// Copyright 2022 OnMetal authors
```

Comment:
Please move this file to `pkg/onmetal/testing/testing.go` as this content won't be used outside.

Reply:
Done